scripts/2.mutation-classifier.py

```diff
@@ -17,27 +18,45 @@
 from dask_searchcv import GridSearchCV
 from sklearn.pipeline import Pipeline, FeatureUnion
 from sklearn.preprocessing import StandardScaler, FunctionTransformer
-from vega import Vega
-import matplotlib.pyplot as plt
 import numpy as np
 import pandas as pd
-import seaborn as sns
+from plotnine import *
```

Comment:
Let's avoid `import *` ([reasons](https://stackoverflow.com/a/2386740/4651668)). If you're worried about plotnine taking up lot's of space you could do something like `import plotnine as pln` or `import plotnine as gg`.